.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build the Python app
+run-name: ${{ github.actor }} - ${{ github.ref_name }}
+
+on:
+    push:
+        branches: ["main", "develop"]
+    pull_request:
+        branches: ["main"]
+
+env:
+    PYTHON_VERSION: 3.13
+
+jobs:
+    python-ci:
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: ./src
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+
+            - name: Setup Python version ${{ env.PYTHON_VERSION }}
+              uses: actions/setup-python@v5
+              with:
+                python-version: '${{ env.PYTHON_VERSION }}'
+
+            - name: Code Style Check
+              run:  |
+                cd ../
+                flake8 . --show-source --statistics --exit-zero
```

Comment:
Using `--exit-zero` with `flake8` means that the command will not fail even if there are style errors. If you want to enforce code style, consider removing this option.